RestApiForGroup2/client.js

```diff
@@ -0,0 +1,123 @@
+var request = require('request');
```

Comment:
Mh.. Is this supposed to be an example use case for group 2? It's confusing... In case, put it in a folder `examples`.

In general: try to keep the repo a bit clearer with namins, aka:

```
/
- scripts
-- scrapers
--- recordings.js
--- releases.js
-- mappings
--- artistByNationality.js
- examples
-- exampleCalls.js
- documentation
-- index.js
-- index.html
-- index.css
```